src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -1,90 +1,232 @@
+.participants__search-icon {
+  position: absolute;
+  height: 100%;
+  width: $icon--medium;
+  color: $color-black;
+  right: $padding--default;
 }
 
-.participants__user-list {
+.participants__filter-buttons {
+  margin-top: $margin--small;
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  border-radius: 8px;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.participants__permisson-filter-button {
+  height: 42px;
   background-color: $color-white;
+  color: $color-black;
+  border: none;
+  border-radius: $rounded--full;
+  padding: 0 10px;
+  font-weight: bold;
+  font-size: 14px;
+  letter-spacing: $letter-spacing--medium;
+  cursor: pointer;
+
+  &--active {
+    background-color: var(--accent-color);
+    color: $color-white;
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+    color: $color-black;
+  }
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+}
+@media screen and (min-width: 920px) {
+  .participants__permisson-filter-button {
+    padding: 0 $padding--default;
+  }
 }
 
-[theme="dark"] {
-  .participants__user-list {
-    background-color: $color-dark-mode-note;
+.participant__status-filter-button {
+  height: 42px;
+  width: 42px;
+  border: none;
+  border-radius: $rounded--full;
+  background-color: $color-white;
+  color: $color-black;
+  padding: 0;
+  cursor: pointer;
+
+  &--active {
+    background-color: var(--accent-color);
+    color: $color-white;
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+    color: $color-black;
+  }
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+  > svg {
+    height: $icon--medium;
+    width: $icon--medium;
   }
 }
 
-.participants__user {
-  position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  padding: 0px $padding--medium;
-  height: 48px;
+.participants__list {
+  margin-top: $margin--default;
+  border-radius: $rounded--default;
+  list-style-type: none;
+  padding: 0;
+  overflow: scroll;
+  margin-bottom: 70px;
 }
 
-.participants__user_avatar-name-wrapper {
+.participants__list-item {
+  height: 55px;
+  background-color: $color-white;
+  padding: $margin--small;
   display: flex;
+  flex-direction: row;
   align-items: center;
-  gap: 16px;
-  position: relative;
+
+  &:not(:last-child) {
+    border-bottom: 1px solid $color-lighter-gray;
+  }
 }
 
-.participants__user_avatar {
-  height: 26px;
-  width: 26px;
+.participant__avatar > svg {
+  height: $icon--extralarge;
+  width: $icon--extralarge;
 }
 
-.participants__user-name {
-  color: $color-black;
-  font-size: $text-size--medium;
-  letter-spacing: $letter-spacing--medium;
-  width: 170px;
+.participant__name-role-wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
   overflow: hidden;
+}
+
+.participant__name {
+  font-size: 14px;
+  font-weight: bold;
+  letter-spacing: $letter-spacing--medium;
+  line-height: 18px;
```

Comment:
The `style.scss` has `$line-height--small` and `$line-height--medium` for 16px and 20px if you want to use them.